content/800-data-platform/250-environments/200-create.mdx

```diff
@@ -228,6 +228,45 @@ The platform introspects the database to read the data models so that it can gen
 
 Your browser is redirected to **Data Browser** for the new environment.
 
+#### If introspection fails and can't generate a schema
+
+You see a notification that introspection failed and cannot generate a schema. This may be because the database is unreachable or run into a timeout. You will see a warning notification at the top you can close after providing a schema.
+
+As a result, [Data Browser](/data-platform/data-browser) and [Query Console](/data-platform/query-console) will be inactive. You can start using them after you provide a schema. You can provide a schema by [uploading your schema](#upload-a-schema-file).
```

Comment:
We need a bit rework here!

 So what happens if introspection fails during environment creation is:
1. User will be prompted an `error` saying introspection failed. In other words, environment was not created
2. User can click `upload a schema` button within the error notification
3. Use can complete the upload process